draft-smyslov-ipsecme-ikev2-downgrade-prevention.xml

```diff
@@ -236,15 +236,16 @@ HDR, SAi1, KEi, Ni,
 
             <figure align="center">
                 <artwork align="left"><![CDATA[
-InitiatorSignedOctets = RealMessage1 | RealMessage2
-                        | NonceRData | MACedIDForI
+InitiatorSignedOctets = ZeroPrefix | RealMessage2 
+                        | RealMessage1 | NonceRData | MACedIDForI
 
-ResponderSignedOctets = RealMessage2 | RealMessage1
-                        | NonceIData | MACedIDForR
+ResponderSignedOctets = ZeroPrefix | RealMessage1 
+					    RealMessage2 | NonceIData | MACedIDForR
             ]]></artwork>
             </figure>
 
-            <t> where RealMessage1, RealMessage2, NonceIData, NonceRData, MACedIDForI and MACedIDForR are defined in Section 2.15 of <xref target="RFC7296" />.
+            <t> where RealMessage1, RealMessage2, NonceIData, NonceRData, MACedIDForI and MACedIDForR are defined in Section 2.15 of <xref target="RFC7296" />,
+				and ZeroPrefix is 8 octets of zero.

```

Comment:
I would add a brief explanation of what `ZeroPrefix` does.

Reply:
I added, but if you have better text in mind, feel free to replace mine.